settings/motorExtensions.cmd

```diff
@@ -0,0 +1,7 @@
+############ Motor Extensions ##############
+# Create a soft motor record on top of a normal motor record which allow the gap to be set for the motor
+# Copy this file to C:\Instrument\Settings\config\<machine>\configurations\galil
+
+$(IFNOTTESTDEVSIM) < $(GALILCONFIG)/xyBeamstop.cmd
+$(IFTESTDEVSIM) < $(MOTOREXTENSIONS)/settings/xyBeamstop/xyBeamstop.cmd
```

Comment:
This is a bit weird. The directory structure for the `*.cmd` files is different in TESTDEVSIM and NOTTESTDEVSIM mode. In the former, the `xyBeamstop.cmd` is in an `xyBeamstop` subdirectory, but in the latter it is at the same level as `motorExtensions.cmd`. This inconsistency is going to cause headache (it did for me!).